util/logging.h

```diff
@@ -16,6 +16,42 @@
 #include <string>
 #include "port/port.h"
 
+// Helper macros that include information about file name and line number
+#define STRINGIFY(x) #x
```

Comment:
Can we add ROCKS_ prefix to these macros just in case customers also using them?